Kitodo-API/src/test/java/org/kitodo/utils/GuardTest.java

```diff
@@ -0,0 +1,184 @@
+package org.kitodo.utils;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.GregorianCalendar;
+
+import org.junit.Test;
+
+public class GuardTest {
```

Comment:
Some notes:
- write a single test method for a single test scenario and not all scenarios in one test method
- exception handling can be done in a lot more readable way. See https://www.baeldung.com/junit-assert-exception for Junit 5 and Junit 4 usage - where I prefer Junit 5

Reply:
- Tests divided.
- I used Junit 5. Yes, I recently discovered that there is this new variant. Much nicer!